src/api/src/services/LogFile/index.ts

```diff
@@ -0,0 +1,36 @@
+import { Service } from 'typedi';
+import { LoggerService } from '../../logger';
+
+export interface ILogFileArgs {
+  numberOfLines: number;
+}
+
+export interface ILogFile {
+  loadLogFile(args: ILogFileArgs): Promise<string>;
+}
+
+@Service()
+export default class LogFileService implements ILogFile {
+  constructor(private logger: LoggerService) {}
+
+  async loadLogFile(args: ILogFileArgs): Promise<string> {
+    const contentPromise = new Promise<string>((resolve) => {
+      this.logger.log('Requested log content');
+      this.logger.query(
+        {
+          rows: args.numberOfLines,
+          order: 'desc',
+          fields: ['timestamp', 'level', 'message', 'context'],
+        },
+        (err, result) => {
+          if (err) {
+            resolve(JSON.stringify({ error: err }));
+          } else {
+            resolve(JSON.stringify(result.file));
```

Comment:
Do you think it would be better to parse log file in the backend? Right now it is being parsed on the UI side.

I think it would be easier to work if this structure was returned by graphql api:

```
export type Log = {
  level: string;
  message: string;
  timestamp: string;
  context?: object;
};
```

Check in `models/` directory how we are modeling graphql entities.

Reply:
Hope i did it right :)